smart-contracts/scripts/lock/samples.js

```diff
@@ -0,0 +1,69 @@
+const { ethers } = require('hardhat')
+const Locks = require('../../test/fixtures/locks')
```

Comment:
This script/task fetch data from test fixtures and deploy the locks locally. This is is very useful for local development